packages/block-editor/src/components/inspector-controls/slot.js

```diff
@@ -1,11 +1,8 @@
 /**
  * WordPress dependencies
  */
-import {
-	__experimentalUseSlotFills as useSlotFills,
-	__unstableMotionContext as MotionContext,
```

Comment:
Now that we removed the last usage, I think this is a good time to remove the unstable `MotionContext` re-export:

https://github.com/WordPress/gutenberg/blob/039bd9081abeb7ce93d7a57e56ca0abfbc6b7937/packages/components/src/animation/index.tsx#L12

https://github.com/WordPress/gutenberg/blob/9defe9f7b6f7147756d1dba26dc0a8f8c07052d0/packages/components/src/index.ts#L27

Looks like there's no usage in plugins:

https://wpdirectory.net/search/01JE66GHXATVJHR1K7PQQG074W

Can be done in a separate PR IMHO.

Reply:
I'm always hesitant about removing APIs, but that should be OK given that there are no usages.

Reply:
#67623